plugins/source/vercel/.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
```

Comment:
Let's remove this and the relevant make target, see https://github.com/cloudquery/cloudquery/pull/5559